frontend/src/app/images/images.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-images',
+  templateUrl: './images.component.html',
+  styleUrls: ['./images.component.css']
+})
+export class ImagesComponent implements OnInit {
+  clickMessage = null;
+  clickMessageTimeout = null;
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  onClickBoy() {
+    clearTimeout(this.clickMessageTimeout);
```

Comment:
nice work adding some real functionality to the app without going too deep. Bonus points :muscle: